.ci/install_qemu.sh

```diff
@@ -77,12 +77,12 @@ install_cached_qemu() {
 
 clone_qemu_repo() {
```

Comment:
Hi @liubin ,

IIUC you added `sudo` calls in `clone_qemu_repo()` because that function is called by  `lib_install_qemu_aarch64.sh` and `lib_install_qemu_ppc64le.sh`. Happens that `clone_qemu_repo()` is also callable by `build_and_install_qemu()` from `install_qemu.sh` when  `FORCE_BUILD_QEMU=yes`. Therefore, `build_and_install_qemu()` should have added the `sudo` calls too.

BTW, I don't know what is the use case for `FORCE_BUILD_QEMU`. It seems none CI job uses it. Anyone has any idea?

Reply:
@devimc what's the use case for `FORCE_BUILD_QEMU` ?

Reply:
I searched `tests` and `ci` repos, not found anywhere used `FORCE_BUILD_QEMU, only aarch64, and ppc64le install QEMU from the source.

And `clone_qemu_repo` only be called in `lib_install_qemu_aarch64.sh`.

Reply:
> I searched `tests` and `ci` repos, not found anywhere used `FORCE_BUILD_QEMU, only aarch64, and ppc64le install QEMU from the source.
> 
> And `clone_qemu_repo` only be called in `lib_install_qemu_aarch64.sh`.

Right but the generic implementation of `build_and_install_qemu()` is also used by s390x. Let's fix this so that the s390x CI  doesn't hit the issue when it is re-enabled. WDYT @liubin ?

Reply:
> > I searched `tests` and `ci` repos, not found anywhere used `FORCE_BUILD_QEMU, only aarch64, and ppc64le install QEMU from the source.
> > And `clone_qemu_repo` only be called in `lib_install_qemu_aarch64.sh`.
> 
> Right but the generic implementation of `build_and_install_qemu()` is also used by s390x. Let's fix this so that the s390x CI doesn't hit the issue when it is re-enabled. WDYT @liubin ?

Fixed, please take a look again.